src/fmu/sumo/explorer/objects/table.py

```diff
@@ -17,45 +18,48 @@ def __init__(self, sumo: SumoClient, metadata: dict) -> None:
             metadata: (dict): child object metadata
         """
         super().__init__(sumo, metadata)
-        self._dataframe = None
+        self._dataframe  = None
         self._arrowtable = None
 
     @property
-    def dataframe(self) -> pd.DataFrame:
+    def to_pandas(self) -> pd.DataFrame:
         """Return object as a pandas DataFrame
 
         Returns:
             DataFrame: A DataFrame object
         """
-        if not self._dataframe:
+        warn('.dataframe() is deprecated, renamed to .to_pandas() ', DeprecationWarning, stacklevel=2)
+        if not self._dataframe :
 
             try:
-                self._dataframe = pd.read_parquet(self.blob)
+                self._dataframe  = pd.read_parquet(self.blob)
```

Comment:
Suggest blacking the code to get rid of these extra spaces and stuff. We are not automatically running black (but we should!).